pkg/process/net/common.go

```diff
@@ -26,10 +26,14 @@ type Conn interface {
 	Stop()
 }
 
+const (
+	contentTypeProtobuf = "application/protobuf"
+)
+
 var (
 	globalUtil            *RemoteSysProbeUtil
 	globalUtilOnce        sync.Once
-	globalSocketPath      string
+	globalPath            string
```

Comment:
`path` isn't really the right name for windows, so I don't think this should be renamed. Let's keep the name as `globalSocketPath`